### PR TITLE
Add radio buttons for user data management options

### PR DESCRIPTION
**Pull request by mla04762**

### CODE REVIEW

**Comment by sharonpatta** [comment]
I would probably change it so that the option says "Delete User Data" instead of "Just Delete User Data"

File: tools/download-json/main.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ type App struct {
 }
 
 func downloadManifest(productId string, result *Manifest) error {
-	manifestUrl := fmt.Sprintf("https://app.scriptoria.io/api/products/%s/files/published/play-listing-manifest", productId)
+	manifestUrl := fmt.Sprintf("https://dev.scriptoria.io/api/products/%s/files/published/play-listing-manifest", productId)
```
Comment (chrisvire) [inline tools/download-json/main.go]: Don't include this change in your commit

Comment (mla04762) [reply tools/download-json/main.go]: I will fix this!

File: src/routes/request/[id]/+page.server.ts [hunk 2]
```diff
@@ -1,10 +1,80 @@
+// src/routes/p/[id]/+page.server.ts
```
Comment (chrisvire) [inline src/routes/request/[id]/+page.server.ts]: Fix comment

File: src/routes/request/[id]/+page.server.ts [hunk 3]
```diff
@@ -1,10 +1,80 @@
...
+            console.error('Request failed. Response: 500 internal server error', response);
+        }
+    } catch (error) {
+        console.error('An error occurred:', error);
+    }
+}
 
 export const load = (async ({ params: { id } }) => {
-    const request = await prisma.userManagementRequest.findUnique({
+    const app = await prisma.userManagementRequest.findUnique({
```
Comment (chrisvire) [inline src/routes/request/[id]/+page.server.ts]: variable should be 'request', not 'app'

File: src/routes/request/[id]/+page.server.ts [hunk 4]
```diff
@@ -1,10 +1,80 @@
...
         where: { id: String(id) },
         include: { app: true }
     });
-    return { request };
+    return { app };
 }) satisfies PageServerLoad;
+
+export const actions = {
+    default: async ({ request, params: { id } }: RequestEvent) => {
+        console.log(SCRIPTORIA_API_TOKEN);
```
Comment (chrisvire) [inline src/routes/request/[id]/+page.server.ts]: Need log?

**Review by chrisvire — CHANGES_REQUESTED** [review]

File: src/routes/request/[id]/+page.svelte [hunk 5]
```diff
@@ -1,9 +1,63 @@
 <script lang="ts">
-    import type { PageData } from './$types';
+    import { onMount } from 'svelte';
 
-    export let data: PageData;
+    // import { API_TOKEN } from '$env/static/private';
```
Comment (chrisvire) [inline src/routes/request/[id]/+page.svelte]: Remove commented out code.

File: src/routes/request/[id]/+page.svelte [hunk 6]
```diff
@@ -1,9 +1,63 @@
...
+    let group = 'user datamanagement';
+    const options = { 'delete-account': 'Delete Account', 'delete-user-data': 'Delete User Data' };
+
+    onMount(() => {
```
Comment (chrisvire) [inline src/routes/request/[id]/+page.svelte]: Is this onMount function needed anymore?

If not needed, remove (and remove import)

**Review by chrisvire — CHANGES_REQUESTED** [review]

File: src/routes/request/[id]/+page.svelte (hunk 6 above)
Comment (mla04762) [reply src/routes/request/[id]/+page.svelte]: the onMount function is needed to make sure that there is at least one option selected.

**Review by chrisvire — APPROVED** [review]
👍

**Pull request merged**